site/src/navigation/sidebar/sidebar-seccondary-nav.less

```diff
@@ -4,19 +4,24 @@ esl-d-sidebar {
   .sidebar-nav-secondary {
     width: 100%;
   }
+
   .sidebar-nav-secondary-list {
     padding: 2px 0;
   }
+
   .sidebar-nav-secondary-item {
     display: block;
     position: relative;
-    padding: 10px 35px 10px 70px;
+    padding-block: 10px;
+    padding-inline-end: 35px;
+    padding-inline-start: var(--padding-inline-start);
```

Comment:
```suggestion
    padding: 10px 35px 10px var(--padding-inline-start, 0);
```

Reply:
Actually, that's not the same thing as padding. The shortcut does not transform into the semantic properties Anna used here.